omod/src/main/webapp/fragments/summary/registrationSummary.gsp

```diff
@@ -40,21 +40,4 @@
             } %>
 
         </div>
-
-        <div class="action-container column">
-            <div class="action-section">
-                <ul>
-                    <h3>${ ui.message("coreapps.clinicianfacing.overallActions") }</h3>
-                    <%
-                        overallActions.each { ext -> %>
-                            <a href="${ ui.encodeJavaScript(ext.url("/" + ui.contextPath(), appContextModel, returnUrl)) }" id="${ ext.id }">
-                                <li>
-                                    <i class="${ ext.icon }"></i>
-                                    ${ ui.message(ext.label) }
-                                </li>
-                            </a>
-                    <% } %>
-                </ul>
-            </div>
-        </div>
```

Comment:
What I said [here](https://issues.openmrs.org/browse/RA-1722?focusedCommentId=262488&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-262488) is that it is ok to not see the _same_ 'General Actions' than those on the clinical dashboard. But I didn't say that the Registration Summary would not allow for the configuration of its own 'General Actions'.

Those `overallActions`, where/how are they fetched?

Reply:
@mks-d am sorry sbout the delay in the replying to this ,unfortunately just seen this ,
the `overallActions `are enabled extensions for the current user from a specified` extensionPointId` and they a bundled to the model from the` RegistrationSummaryFragmentController`